WoWsShipBuilder.Web/Components/ShipStatsContainer.razor

```diff
@@ -142,15 +148,30 @@
 
     private bool shipChanged;
 
-    private string buildName = string.Empty;
+    private string BuildName
+    {
+        get => CacheService[CurrentTabId]?.BuildName ?? string.Empty;
```

Comment:
Feel free to use the ! operator here. Once this property is accessed, that value must not be null anyways